src/ui/src/builder/BuilderFieldsKeyValue.vue

```diff
@@ -38,16 +43,16 @@
 				</div>
 			</div>
 			<div class="formAdd">
-				<input
+				<BuilderTemplateInput
 					ref="assistedKeyEl"
-					v-model="formAdd.key"
-					type="text"
+					v-model:value="formAdd.key"
```

Comment:
To use `v-model` with `BuilderTemplateInput` you'll need to use `defineModel` (at least that's what's documented)